### DISCUSSION TITLE
抖音课堂

### DISCUSSION THREAD

**Discussion by fjxmjrr** [PyJun/Mooc_Downloader#152] [Q&A]
抖音课堂的音频课程下载好打开播放是没有声音的

**Comment by PyJun** [comment]
建议使用potplayer播放器，再试试

**Comment by fjxmjrr** [comment]
就是用这个播放器的



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年10月18日(周三) 中午12:57
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152)




 
建议使用potplayer播放器，再试试

**Comment by PyJun** [comment]
重下试试，不行就反馈公共号客服



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年10月18日(周三) 中午1:07
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152)




 就是用这个播放器的 
 
 
 
 ---原始邮件--- 
 发件人: ***@***.***&amp;gt; 
 发送时间: 2023年10月18日(周三) 中午12:57 
 收件人: ***@***.***&amp;gt;; 
 抄送: ***@***.******@***.***&amp;gt;; 
 主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152) 
 
 
 
 
   
 建议使用potplayer播放器，再试试

**Comment by fjxmjrr** [comment]
没有币可下了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年10月18日(周三) 中午1:09
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152)




 重下试试，不行就反馈公共号客服 
 
 
 
 ---原始邮件--- 
 发件人: ***@***.***&amp;gt; 
 发送时间: 2023年10月18日(周三) 中午1:07 
 收件人: ***@***.***&amp;gt;; 
 抄送: ***@***.******@***.***&amp;gt;; 
 主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152) 
 
 
 
 
  就是用这个播放器的  
   
   
   
  ---原始邮件---  
  发件人: ***@***.***&amp;amp;gt;  
  发送时间: 2023年10月18日(周三) 中午12:57  
  收件人: ***@***.***&amp;amp;gt;;  
  抄送: ***@***.******@***.***&amp;amp;gt;;  
  主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152)  
   
   
   
   
     
  建议使用potplayer播放器，再试试

**Comment by fjxmjrr** [comment]
如果充值，再下载还是没有声音怎么办



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年10月18日(周三) 中午1:09
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152)




 重下试试，不行就反馈公共号客服 
 
 
 
 ---原始邮件--- 
 发件人: ***@***.***&amp;gt; 
 发送时间: 2023年10月18日(周三) 中午1:07 
 收件人: ***@***.***&amp;gt;; 
 抄送: ***@***.******@***.***&amp;gt;; 
 主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152) 
 
 
 
 
  就是用这个播放器的  
   
   
   
  ---原始邮件---  
  发件人: ***@***.***&amp;amp;gt;  
  发送时间: 2023年10月18日(周三) 中午12:57  
  收件人: ***@***.***&amp;amp;gt;;  
  抄送: ***@***.******@***.***&amp;amp;gt;;  
  主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152)  
   
   
   
   
     
  建议使用potplayer播放器，再试试

**Comment by PyJun** [comment]
有问题反馈公众号“学无止科技”客服



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年10月22日(星期天) 中午12:54
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152)





 如果充值，再下载还是没有声音怎么办 
 
 
 
 ---原始邮件--- 
 发件人: ***@***.***&amp;gt; 
 发送时间: 2023年10月18日(周三) 中午1:09 
 收件人: ***@***.***&amp;gt;; 
 抄送: ***@***.******@***.***&amp;gt;; 
 主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152) 
 
 
 
 
  重下试试，不行就反馈公共号客服  
   
   
   
  ---原始邮件---  
  发件人: ***@***.***&amp;amp;gt;  
  发送时间: 2023年10月18日(周三) 中午1:07  
  收件人: ***@***.***&amp;amp;gt;;  
  抄送: ***@***.******@***.***&amp;amp;gt;;  
  主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152)  
   
   
   
   
   就是用这个播放器的   
     
     
     
   ---原始邮件---   
   发件人: ***@***.***&amp;amp;amp;gt;   
   发送时间: 2023年10月18日(周三) 中午12:57   
   收件人: ***@***.***&amp;amp;amp;gt;;   
   抄送: ***@***.******@***.***&amp;amp;amp;gt;;   
   主题: Re: [PyJun/Mooc_Downloader] 抖音课堂 (Discussion #152)   
     
     
     
     
       
   建议使用potplayer播放器，再试试
